### ISSUE TITLE
What does appropriate set of protections mean? 

### ISSUE THREAD

**Issue by dig-comments** [usnistgov/800-63-3#1417]
**Organization Name (N/A, if individual)**: CMS

**Organization Type**: 1
 
**Document (63-3, 63A, 63B, or 63C)**: 800-63C

**Reference (Include section and paragraph number)**: Section 5.2 (Assertion Protection),
Para 1
, 
 
**Comment (Include rationale for comment)**:  What does appropriate set of protections mean?
 
This is a subjective term where examples will help to ensure the implementer will be better equipped to understand the term in an objectivity manner.

**Suggested Change**: 
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by dig-comments** [comment]
@jricher Did we mean in the intro paragraph 'the following protections shall be used'? Doesn't seem like it since we have PPI in the list, and those are optional, but wanted to check before I accepted.

**Comment by jimfenton** [comment]
Reword the paragraph to introduce requirements that follow. Consider moving pairwise identifier sections to a separate section (5.3.x?)

**Comment by jricher** [comment]
Agree that current wording is awkward. The required protections are dependent on the use case, which is expanded more in the FAL section. I'd suggest rewording the opening paragraph to something like:

> Independent of the binding mechanism (discussed in Section 5.1) or the federation model used to obtain them (described in Section 4), assertions SHALL include a set of protections to prevent attackers from manufacturing valid assertions or reusing captured assertions at disparate RPs. The protections required are dependent on the details of the use case being considered, and recommended protections are listed here.
